src/test/resources/expected/expectedBuyFruitAfterReturnOk.csv

```diff
@@ -0,0 +1,2 @@
+fruit,quantity
+apple,25
```

Comment:
add an empty line at the end of all files

Reply:
done